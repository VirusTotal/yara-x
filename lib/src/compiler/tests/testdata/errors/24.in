rule test {
  strings:
    $a = "foo"
  condition:
    "1" of them
}