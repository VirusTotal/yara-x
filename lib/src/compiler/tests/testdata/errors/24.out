error: wrong type
 --> line:5:5
  |
5 |     "1" of them
  |     ^^^ expression should be `integer`, but is `string`
  |